### ISSUE TITLE
Setting document root in lighttpd.conf

### ISSUE THREAD

**Issue by abhivachan** [jacobsalmela/pi-hole#125]
Not sure the purpose of changing the below setting in lighttpd.  

from 
server.document-root        = "/var/www"
to
server.document-root        = "/var/www/html"

It is breaking other sites configured in lighttpd. I have installed phpmyadmin, nagios3, nconf and configured the same for lighttpd.

**Comment by jacobsalmela** [comment]
The latest version of `lighttpd` in Jessie moved the default root to `/var/www/html`.  So we would only need one config file I [opted to change it](https://github.com/jacobsalmela/pi-hole/issues/50) for both Wheezy and Jessie.  

Most of my user base (based on comments and emails I get everyday) is using the Pi as a **standalone** Pi-hole, so this hasn't been much of an issue.  However, I also acknowledge that there are also plenty of tinkerers and customizers out there.

But as a lone, part-time developer, it's been getting more and more difficult to meet everyone's needs, especially as the project has grown quite a bit.  I basically write the code for fun, and then I do a big development push when some donations come in.  My biggest focus is letting users take a new Raspberry Pi, install the Pi-hole, start blocking ads, and then forget about it.

I am working a lot with @dschaper to Debianize the project as well as make it portable to other OSes, while not getting in the way of other stuff people have running.  

Bottom line: I don't think there will be much I can do at this point unless I could work full-time on the Pi-hole or some other developers feel like stepping up. Sorry.
